examples/jsm/loaders/GLTFLoader.js

```diff
@@ -3879,18 +3887,91 @@ class GLTFParser {
 	 */
 	loadNode( nodeIndex ) {
 
+		const json = this.json;
+		const parser = this;
+
+		const nodeDef = json.nodes[ nodeIndex ];
+
+		return ( function () {
+
+			const nodePending = parser._loadNode( nodeIndex );
+
+			const childPending = [];
+			const childrenDef = nodeDef.children || [];
+
+			for ( let i = 0, il = childrenDef.length; i < il; i ++ ) {
+
+				childPending.push( parser.getDependency( 'node', childrenDef[ i ] ) );
+
+			}
+
+			const skeletonPending = nodeDef.skin === undefined
+				? Promise.resolve( null )
+				: parser.getDependency( 'skin', nodeDef.skin );
+
+			return Promise.all( [
+				nodePending,
+				Promise.all( childPending ),
+				skeletonPending
+			] );
+
+		}() ).then( function ( results ) {
```

Comment:
This IIFE closure seems to have been introduced at some point for a certain reason but I couldn't track.

I agree with wriging without IIFE closure because it would be more readable. But do you think it's ok for you to do that in another PR? It will be easier to revert if this IIFE closure is really needed for something special reasons.